radix-engine-queries/src/typed_substate_layout.rs

```diff
@@ -465,7 +465,7 @@ pub enum TypedMainModuleSubstateValue {
     PackageSchema(KeyValueEntrySubstate<ScryptoSchema>),
     PackageCode(KeyValueEntrySubstate<PackageCodeSubstate>),
     PackageAuthTemplate(KeyValueEntrySubstate<AuthConfig>),
-    PackageRoyalty(KeyValueEntrySubstate<ComponentRoyaltyConfig>),
+    PackageRoyalty(KeyValueEntrySubstate<PackageRoyaltyConfig>),
```

Comment:
Ohh nice, thanks for the fix. Glad to see the scenario => typed substates is paying off